### PR TITLE
MDM-12146

### PR DESCRIPTION
**Pull request by kkeskinsoy**
move the menu items debug and logger from the main menu to the project menu,
bug fix: remove double slash in url

### CODE REVIEW

File: src/commands/Project.ts [hunk 1]
```diff
@@ -60,6 +62,38 @@ export class Project extends Command {
...
+      },
+      description: this.i18n.DEBUGGER_OPEN_DESCRIPTION,
+      vars: {
+        server: {
+          pos: 0
+        }
+      }
+    },
+    log: {
+      label: 'logger',
```
Comment (t-beckmann) [inline src/commands/Project.ts]: I wonder why the function is named log instead of logger like the menu item is named. Is there a reason for this?

**Comment by DrMabuse23** [comment]
@kkeskinsoy 
```shell
Relution-Cli v0.0.26: 
Hi pascalbrewing
? Please Choose Your Option:  project
? Please Choose Your project Command:  logger
? Select a Server Cancel
Pascals-MacBook-Pro:rel-cli-test pascalbrewing$ 
````

normally after Cancel you have to go back to menu Project not process.exit()

File: src/commands/project/Debugger.ts [hunk 2]
```diff
@@ -1,47 +1,38 @@
...
+import { DebugLog } from '../../utility/DebugLog';
+import { Translation } from '../../utility/Translation';
+import { RelutionSdk } from '../../utility/RelutionSDK';
+import { RxFs } from './../../utility/RxFs';
 
-export class Debugger extends Command {
+export class Debugger {
 
-  private _deployCommand: Deploy;
+  public _deployCommand: Deploy;
```
Comment (DrMabuse23) [inline src/commands/project/Debugger.ts]: @kkeskinsoy a public var has no underscore

File: src/commands/project/Debugger.ts [hunk 3]
```diff
@@ -1,47 +1,38 @@
...
   private _relutionHjson: any;
   private _fileApi: FileApi = new FileApi();
+  private owner: Command;
```
Comment (DrMabuse23) [inline src/commands/project/Debugger.ts]: @kkeskinsoy private var has to begin with _ please check your tslint

**Pull request merged**